conjur/cli.py

```diff
@@ -737,7 +739,8 @@ def run_action(resource, args):
         if not is_testing_env:
             loaded_conjurrc = ConjurrcData.load_from_file()
             if not credential_provider.is_exists(loaded_conjurrc.conjur_url):
-                sys.stdout.write("%s\n" % LOGIN_IS_REQUIRED)
+                # pylint: disable=logging-fstring-interpolation
+                sys.stdout.write(f"{LOGIN_IS_REQUIRED}\n")
                 Cli.handle_login_logic(credential_provider, ssl_verify=args.ssl_verify)
 
```

Comment:
There are two places in the KeyringAdapter that do not have tests for. Do you see value in adding tests for this as well?

![Screen Shot 2021-03-31 at 12 57 00 PM](https://user-images.githubusercontent.com/19418506/113126970-e3991000-9220-11eb-99d2-2cfb3928c681.png)